ocaml/idl/datamodel_vm.ml

```diff
@@ -1088,8 +1088,11 @@ let power_behaviour =
       ~name:"set_suspend_VDI"
       ~in_product_since:rel_boston
       ~doc:"Set this VM's suspend VDI, which must be indentical to its current one"
-      ~params:[Ref _vm, "self", "The VM";
-               Ref _vdi, "value", "The suspend VDI uuid"]
+      ~versioned_params:
+          [{param_type=Ref _vm; param_name="self"; param_doc="The VM"; param_release=boston_release; param_default=None};
+           {param_type=Ref _vdi; param_name="value"; param_doc="The suspend VDI uuid"; param_release=boston_release; param_default=None};
+           {param_type=Bool; param_name="ignore_checksum"; param_doc="Ignore checksum or not."; param_release=tampa_release; param_default=Some (VBool false)}
```

Comment:
IMHO, it would be better to simply allow a `suspend_VDI` field in `VM.create` in case `power_state` is `Suspended`.

This would limit this special case at VM creation and would not require changing `set_suspend_VDI`.

Reply:
Agreed. It should be an error to try and create a not-suspended VM with a non-null `suspend_VDI`, though.